src/Zypper.cc

```diff
@@ -555,7 +555,7 @@ void Zypper::doCommand( int cmdArgc, char **cmdArgv, int firstFlag )
             repo.setName( url.asString() );
 
             repo.setMetadataPath( runtimeData().tmpdir / repo.alias() / "%AUTO%" );
-            repo.setPackagesPath( Pathname::assertprefix( _config.root_dir, ZYPPER_RPM_CACHE_DIR ) );
+            repo.setPackagesPath( Pathname::assertprefix( _config.root_dir, _config.rm_options.repoPackagesCachePath ) / "@System" );
```

Comment:
Maybe a comment that @System was chosen because libzypp won't autoclean it (i.e. a different dirname won't work without touching libzypp)

Reply:
Done